src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -474,3 +474,31 @@ bool Terminal::SetDefaultBackground(const COLORREF dwColor)
     _buffer->GetRenderTarget().TriggerRedrawAll();
     return true;
 }
+
+// Method Description:
+// - Saves the current text attributes to an internal stack.
+// Arguments:
+// - options, cOptions: if present, specify which portions of the current text attributes
+//   should be saved. Only a small subset of GraphicsOptions are actually supported;
+//   others are ignored. If no options are specified, all attributes are stored.
+// Return Value:
+// - true
+bool Terminal::PushGraphicsRendition(const ::Microsoft::Console::VirtualTerminal::DispatchTypes::GraphicsOptions* options, size_t cOptions)

```

Comment:
So it's implemented in the terminal still, too. Just in case it's attached to a different source that isn't a ConPTY-based source where conhost handles it. Right?

Reply:
I did it this way based on initial guidance from zadjii-msft... sounds right.